docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+  violated the deadline contract concerning the data instances.
+  That `strength` can be changed using the :ref:`ownershipstrengthqospolicy`.
+
+.. _ownership_compatibilityrule:
+
+Compatibility Rule
+""""""""""""""""""
+
+To maintain the compatibility between OwnershipQosPolicy in DataReaders and DataWriters, the DataWriter kind must be
+equal to the DataReader kind.
+
+Table with the possible combinations:
+
++-----------------------+-----------------------+-----------------+
+| DataWriter kind       | DataReader kind       | Compatibility   |
++=======================+=======================+=================+
+| Shared                | Shared                | Yes             |
++-----------------------+-----------------------+-----------------+
+| Shared                | Exclusive             | No              |
++-----------------------+-----------------------+-----------------+
+| Exclusive             | Shared                | No              |
++-----------------------+-----------------------+-----------------+
+| Exclusive             | Exclusive             | Yes             |
++-----------------------+-----------------------+-----------------+
+
+Example
+"""""""
+
+C++
+***
+.. literalinclude:: ../../../../../code/DDSCodeTester.cpp
+   :language: c++
+   :dedent: 8
+   :start-after: //DDS_CHANGE_OWNERSHIP_QOS_POLICY
+   :end-before: //!
+
+XML
+***
+This QoS Policy cannot be configured using XML by the moment.
+
+.. _ownershipstrengthqospolicy:
+
+OwnershipStrengthQosPolicy
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This QoS Policy specifies the value of the `strength` used to arbitrate among multiple DataWriters that attempt to
+modify the same data instance. It is only applicable if the :ref:`ownershipqospolicy` kind is set to EXCLUSIVE.
+
+List of QoS Policy data members:
+
++--------------------------+-------------------------------+-------------------+
+| Data Member Name         | Type                          | Default Value     |
++==========================+===============================+===================+
+| value                    | uint32_t                      | 0                 |
++--------------------------+-------------------------------+-------------------+
+
+.. note::
+     This QoS Policy concerns to DataWriter entities.
+     :raw-html:`<br />`
+     It can be changed after the creation of the entities.
+
+Example
+"""""""
+
+C++
+***
+.. literalinclude:: ../../../../../code/DDSCodeTester.cpp
+   :language: c++
+   :dedent: 8
+   :start-after: //DDS_CHANGE_OWNERSHIP_STRENGTH_QOS_POLICY
+   :end-before: //!
+
+XML
+***
+This QoS Policy cannot be configured using XML by the moment.
+
+.. _partitionqospolicy:
+
+PartitionQosPolicy
+^^^^^^^^^^^^^^^^^^
+
+This Qos Policy allows the introduction of a logical partition inside the physical partition introduced by a domain.
+For a DataReader to see the changes made by a DataWriter, not only the Topic must match, but also they have to be on
+the same logical partition.
```

Comment:
```suggestion
For a DataReader to see the changes made by a DataWriter, not only the Topic must match, but also they have to share
at least one logical partition.
```
I think the old wording may induce the user to think the complete value of the partition qos must be the same in both endpoints